### PR TITLE
NLS messages for one or more repositories

### PR DESCRIPTION
**Pull request by njr-11**
Some of the NLS message for Jakarta Data were written before we were informed of the correct way to refer to the possibility of having one or more.  This PR goes back and updates those prior messages (all of which are new and have not gone into a release yet) to cover the possibility of one or more.

- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Comment by njr-11** [comment]
#build

**Review by KyleAure — DISMISSED** [review]

File: dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops [hunk 1]
```diff
@@ -633,24 +633,24 @@ CWWKD1064.datastore.error.explanation=An error occurred when attempting to obtai
...
  within {2} seconds for the following entities: {3}.
 CWWKD1065.ddlgen.timeout.explanation=The DDL generation attempt timed out \
  and did not complete.
 CWWKD1065.ddlgen.timeout.useraction=Reattempt DDL generation \
  when the system and database are not under load.
 
-CWWKD1066.ddlgen.failed=CWWKD1066E: DDL generation for the {0} repository \
- interface that uses the {1} data store encountered an error when attempting \
+CWWKD1066.ddlgen.failed=CWWKD1066E: DDL generation for one or more repository \
+ interfaces ({0}) that use the {1} data store encountered an error when attempting \
```
Comment (dmuelle) [inline dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops]:
```suggestion
 interfaces ({0}) that use the {1} data store encountered an error during \
```

File: dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops [hunk 2]
```diff
@@ -734,8 +736,8 @@ CWWKD1076.repo.disposed.useraction=Ensure that the repository method is not \
  used outside of the scope of the application that defines the repository.
 
 CWWKD1077.defaultds.not.found=CWWKD1077E: The {0} application artifact \
- has a {1} repository interface that uses or defaults to the \
- {2} data store, which must be configured in server configuration \
+ has one or more repository interfaces ({1}) that use or default to the \
+ {2} data store. This data store must be configured in server configuration \
```
Comment (dmuelle) [inline dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops]:
```suggestion
 {2} data store. This data store must be configured in the server configuration \
```

**Review by dmuelle — COMMENTED** [review]
ID review- a couple minor suggestions

**Comment by njr-11** [comment]
> ID review- a couple minor suggestions

Thanks for reviewing!  I included both of your suggestions as commits.  Please add your approval if everything looks good.

**Comment by njr-11** [comment]
#build
#libby

**Review by dmuelle — DISMISSED** [review]

**Review by KyleAure — APPROVED** [review]

**Comment by njr-11** [comment]
The build failed because a unit test was not updated to account for the method signature change that was needed to supply the parameters for some of the NLS messages. It is updated now, so retrying another build:
#build
#libby

**Comment by njr-11** [comment]
> https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?uuid=3c45e8ae-eb86-43a6-b932-85f7ad197267 completed successfully!

Using this build

**Pull request merged**